llvm/lib/Target/AMDGPU/SIFixSGPRCopies.cpp

```diff
@@ -124,6 +124,12 @@ class SIFixSGPRCopies {
   SmallVector<MachineInstr*, 4> RegSequences;
   SmallVector<MachineInstr*, 4> PHINodes;
   SmallVector<MachineInstr*, 4> S2VCopies;
+  struct V2PysSCopyInfo {
+    SmallVector<MachineOperand *> MOs;
+    SmallVector<Register> SGPRs;
+  };
+  DenseMap<MachineInstr *, struct V2PysSCopyInfo> WaterFalls;
```

Comment:
You can omit the struct keyword.
```suggestion
  DenseMap<MachineInstr *, V2PysSCopyInfo> WaterFalls;
```